### ISSUE TITLE
Paladin - Permanent Aura mastery

### ISSUE THREAD

**Issue by kysqt** [dalaranwow/dalaran-wow#5216]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: For some reason Aura mastery can become permanent I have no idea how to recreate this bug but it's been a common thing on this server for a while, I just did some arena games and ran into a paladin with this bug, completely immune to my Wind Shear. 

Here is a video from last year https://www.youtube.com/watch?v=a8KbZaq_Uac&list= you can clearly see no aura mastery is up yet he cannot be interupted.

**Expected behaviour**: Aura Mastery should not have a 100% up time.

**Steps to reproduce the problem**:

1. No idea
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
moved to https://github.com/dalaranwow/dalaran-wow/issues/5343
